azkaban-exec-server/src/main/java/azkaban/execapp/FlowRunnerManager.java

```diff
@@ -260,6 +260,13 @@ public boolean accept(File dir, String name) {
     return allProjects;
   }
 
+  private boolean isActive() throws ExecutorManagerException {
```

Comment:
So this method is not used yet. Right? Would it be better to include that change in the next pull request then so that they are together?